src/es/components/atoms/Logo.js

```diff
@@ -141,6 +143,11 @@ export default class Logo extends Shadow() {
         text-decoration: var(--text-a-text-decoration-hover, none);
       }
       @media only screen and (max-width: ${this.getAttribute('mobile-breakpoint') ? this.getAttribute('mobile-breakpoint') : self.Environment && !!self.Environment.mobileBreakpoint ? self.Environment.mobileBreakpoint : '1000px'}) {
+        :host {
+          position: var(--position-mobile, static);
+          align-self: var(--align-self-mobile, center);
```

Comment:
instead of align self you may can play with the https://developer.mozilla.org/en-US/docs/Web/CSS/order